### DISCUSSION TITLE
Sending data to third-party

### DISCUSSION THREAD

**Discussion by JohannesAlmroth-WHO-UMC** [stryker-mutator/stryker-net#3115] [Q&A] [answered]
Hi! I didn't see this explicitly mentioned anywhere, so I just wanted to ask; does any part of the stryker toolset send data to a third party, or is everything confined to the machine that runs it? I'm also thinking about the azure devops extension that generates test reports in the pipeline run.

Sorry if this has been mentioned elsewhere, just wanted to check.

**Comment by rouke-broersma** [comment, marked as answer]
Hi Johannes, stryker performs a version check against nuget.org on startup, you can disable this check with a command line argument. This is the only 3rd party network call performed by Stryker by default.

The azure devops plugin saves the report state to your azure devops pipeline storage. I don't know if you classify this as 3rd party, but since this data is generated by your azure devops and then stored in your azure devops I don't consider this third party from the perspective of the extension.

You can optionally store your report in the Stryker dashboard (https://dashboard.stryker-mutator.io) or in azure filesharing if you both enable those reporters in Stryker and supply appropriate credentials for those services.

**Reply by JohannesAlmroth-WHO-UMC** [reply]
Thanks for the quick and explanatory answer. 👍
